apps/challenges/aws_utils.py

```diff
@@ -615,6 +616,115 @@ def scale_workers(queryset, num_of_tasks):
     return {"count": count, "failures": failures}
 
 
+def scale_resources(challenge, new_cpu_units, new_memory):
```

Comment:
can you update the variable names to `worker_cpu_cores` and `worker_memory`

Reply:
For vCPU count, why are they currently named "cpu_cores", considering that the vCPU count given to AWS does not correspond to actual virtual or physical cores available to the worker?